packages/tasks/src/tasks/feature-extraction/spec/output.json

```diff
@@ -0,0 +1,51 @@
+{
+	"$id": "/inference/schemas/feature-extraction/output.json",
+	"$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
This one was not exported I think

Reply:
Yes - it's an issue with the package we use for code generation, I don't know how to fix it yet